examples/tree_2d_dgsem/elixir_navierstokes_taylor_green_vortex.jl

```diff
@@ -0,0 +1,75 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Navier-Stokes equations
+
+# TODO: parabolic; unify names of these accessor functions
+prandtl_number() = 0.72
+mu() = 6.25e-4 # equivalent to Re = 1600
+
+equations = CompressibleEulerEquations2D(1.4)
+equations_parabolic = CompressibleNavierStokesDiffusion2D(equations, mu=mu(),
+                                                          Prandtl=prandtl_number())
+
+"""
+    initial_condition_taylor_green_vortex(x, t, equations::CompressibleEulerEquations2D)
+
+The classical inviscid Taylor-Green vortex, see for instance 
```

Comment:
You are considering the viscous case, aren't you?

Reply:
In principle yes, since we have the parabolic terms from Navier-Stokes.

I basically adapted this example from the existing in 3D

https://github.com/trixi-framework/Trixi.jl/blob/73e58dc59ad0e06616507b9338c8fe0bee5b99b4/examples/tree_3d_dgsem/elixir_navierstokes_taylor_green_vortex.jl#L16-L33

which does not include the damping due to viscosity although we use the Navier-Stokes equations.

IMO, it would make sense to adapt both examples to the viscous case and, if desired, add the inviscid case with Euler Equations only?

Reply:
It's fine to keep the other examples - but please adapt the docstring here since it's not the inviscid case